tests/std/tests/P0645R10_text_formatting_parsing/test.cpp

```diff
@@ -156,6 +168,38 @@ constexpr bool test_parse_arg_id() {
     return true;
 }
 
+template <typename CharT>
+constexpr bool test_parse_precision() {
+    auto parse_pre_fn = _Parse_precision<CharT, testing_callbacks<CharT>>;
+    using view_typ    = basic_string_view<CharT>;
+
+    auto s0 = view_typ(TYPED_LITERAL(CharT, ".0"));

```

Comment:
Our test code style tends to be "relaxed", but I observe that this is using the `auto` pattern that I find unnecessarily verbose and confusing: constructing temporaries and using `auto`, instead of just directly typing the objects.

For example, I believe these lines should be:

```cpp
view_typ s0(TYPED_LITERAL(CharT, ".0"));
```

Reply:
ah you're right, likely a result of several revisions to try and figure out how to best do the whole TYPED_LITERAL thing

Reply:
change incoming